wolfcrypt/src/asn.c

```diff
@@ -14438,22 +14589,26 @@ static void AddAltName(DecodedCert* cert, DNS_entry* dnsEntry)
 #endif
 
 #ifdef WOLFSSL_ASN_TEMPLATE
-#ifdef WOLFSSL_SEP
+#if defined(WOLFSSL_SEP) || defined(WOLFSSL_FPKI)
 /* ASN.1 template for OtherName of an X.509 certificate.
  * X.509: RFC 5280, 4.2.1.6 - OtherName (without implicit outer SEQUENCE).
  * HW Name: RFC 4108, 5 - Hardware Module Name
  * Only support HW Name where the type is a HW serial number.
```

Comment:
Add to the comment to indicate that what UPN and FASC-N are and how one of these will appear.